package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Brave laptop and desktop browser",
   "main": "./app/index.js",
   "config": {
-    "port": "8080"
+    "port": "8080",
+    "electron_mirror": "https://github.com/brave/electron/releases/download/v",
```

Comment:
to be honest I"m not sure where browser-laptop-releases is configured to be used, but I think it makes it obsolete either way